ca/server.go

```diff
@@ -427,7 +440,19 @@ func (s *Server) Run(ctx context.Context) error {
 	s.ctx, s.cancel = context.WithCancel(ctx)
 	ctx = s.ctx
 	close(s.started)
+	// we need to set it on the server, because `Server.UpdateRootCA` can be called from outside the Run function
+	s.rootReconciler = newReconciler(
+		log.WithField(ctx, "method", "(*Server).rootRotationReconciler"),
+		s.securityConfig.ClientTLSCreds.Organization(),
+		s.rootReconciliationRetryInterval,
+		s.store, s.lastSeenClusterRootCA, nodes)
```

Comment:
I'd prefer to just inline the contents of `newReconciler` here. Passing so many things through function arguments loses clarity over initializing a structure directly.